### DISCUSSION TITLE
How to handle #import directives?

### DISCUSSION THREAD

**Discussion by selassje** [microsoft/vscode-cpptools#7039] [Q&A] [answered]
Intellisense is not finding the tlh file, even though it's in the include path

**Comment by sean-mcmanus** [comment, marked as answer]
What compilerArg's are you using for cl.exe. I assume /clr and some others?

**Reply by selassje** [reply]
None. I am building for native so should I use /clr?

**Reply by sean-mcmanus** [reply]
Oh, so this usage of `#import` is for a COM library (https://docs.microsoft.com/en-us/cpp/preprocessor/hash-import-directive-cpp?view=msvc-160)? I think this is the first request we've had for that. I don't think we've implemented support for that yet.

**Reply by selassje** [reply]
Yes, it is COM.

**Reply by sean-mcmanus** [reply]
I've filed a feature request at https://github.com/microsoft/vscode-cpptools/issues/7062
